development/proposal_template.md

```diff
@@ -0,0 +1,28 @@
+## The Current Situation
```

Comment:
This looks like it was copied out of a pdf.  It needs to be written from our perspective using our words.